dir.props

```diff
@@ -71,6 +51,11 @@
     <PortableBuild Condition="'$(PortableBuild)' == ''">true</PortableBuild>
 
     <OverrideRestoreOutputPath>true</OverrideRestoreOutputPath>
+
+    <!-- If true, indicates that this is not an officially supported release -->
+    <!-- It is important to flip this to false in official release branches -->
+    <!-- Keep it in sync with PRERELEASE in clrdefinitions.cmake -->
+    <IsPrerelease>true</IsPrerelease>
```

Comment:
Can we add this to Directory.Build.props if not already there.

Reply:
I agree this should be in a `Directory.Build.props` files, but in the `src/` one not in the root of the repo. Tests use the root one as well as shouldn't be polluted by that setting.